packages/agent/src/certificate.ts

```diff
@@ -122,6 +120,17 @@ function isBufferEqual(a: ArrayBuffer, b: ArrayBuffer): boolean {
   return true;
 }
 
+function isBufferGreaterThan(a: ArrayBuffer, b: ArrayBuffer): boolean {
```

Comment:
Can use `compare` instead.

`isBufferGreaterThan` is equivalent to `compare(a, b) > 0`

Reply:
Nice catch. I'll also remove the `isBufferEqual` function in this file in favor of the common utils